active/0000-standarise-targets.md

```diff
@@ -0,0 +1,152 @@
+- Start Date: 2017-01-03
+- RFC PR: (leave this empty)
+- Ember CLI Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes the introduction of a official convention to specify the targets of an
+application.
+
+### Motivation
+
+Javascript and the platforms it runs on are moving targets. NodeJS and browsers release new
+versions every few weeks. Browsers auto update and each update brings new language features
+and APIs.
+
+Developers need an easy way to express intention that abstracts them from the ever-changing
+landscape of versions and feature matrices, so this RFC proposes to introduce a universal
+configuration option to let developers express they intended targets
+
+This configuration should be easily available to addons, but this RFC doesn't impose
+any mandatory behavior on addons. All addons that want to customize their behavior
+depending on the target platforms will have a single source of truth to get that
+information but it's up to them how to use it.
+
+The advantage of having a single source of truth for the targets compared to configure this
+on a per-addon basis like we do today is mostly for better ergonomics and across-the-board
+consistency.
+
+Addons that would benefit from this conventions are `babel-preset-env`, `autoprefixer`,
+`stylelint` and `eslint` (vía `eslint-plugin-compat`), but not only those. Even Ember itself could,
+once converted into an addon, take advantage of that to avoid polyfilling or even taking
+advantage of some DOM API (`node.classList`?) deep in Glimmer's internals.
+
+### Detailed implementation
```

Comment:
It wasn't blank, it's just that it was an h3 and it should be an h1, and the next points are subsections.